asyncpg/protocol/protocol.pyx

```diff
@@ -353,6 +353,9 @@ cdef class BaseProtocol(CoreProtocol):
                         # Abort the COPY operation on any error in
                         # output sink.
                         self._request_cancel()
+                        # Make asyncio shut up about unretrieved
+                        # QueryCanceledError
+                        waiter.add_done_callback(lambda f: f.exception())
```

Comment:
Please make sure you have a warning here, as we fixed that stale cancellation exception warning in 3.6.2

Reply:
This is not actually about `asyncio.CancelledError`, this is `asyncpg.QueryCanceledError`, which now propagates when an operation is cancelled.